applications/rag/frontend/variables.tf

```diff
@@ -66,4 +66,86 @@ variable "google_service_account" {
   type        = string
   description = "Google Service Account name"
   default = "frontend-gcp-sa"
+}
+
+variable "add_auth" {
+  type        = bool
+  description = "Enable iap authentication on frontend"
+  default     = true
+}
+
+variable "k8s_ingress_name" {
+  type    = string
+  default = "frontend-ingress"
+}
+
+variable "k8s_managed_cert_name" {
+  type          = string
+  description   = "Name for frontend managed certificate"
+  default       = "frontend-managed-cert"
+}
+
+variable "k8s_iap_secret_name" {
+  type    = string
+  default = "frontend-secret"
+}
+
+variable "k8s_backend_config_name" {
+  type        = string
+  description = "Name of the Backend Config on GCP"
+  default     = "frontend-iap-config"
+}
+
+variable "k8s_backend_service_name" {
+  type        = string
+  description = "Name of the K8s Backend Service, this is defined by Frontend"
+  default     = "rag-frontend"
+}
+
+variable "k8s_backend_service_port" {
+  type        = number
+  description = "Name of the K8s Backend Service Port"
+  default = 8080
+}
+
+variable "brand" {
+  type        = string
+  description = "name of the brand if there isn't already on the project. If there is already a brand for your project, please leave it blank and empty"
+  default     = ""
+}
+
+variable "url_domain_addr" {
+  type        = string
+  description = "Domain provided by the user. If it's empty, we will create one for you."
+  default     = ""
+}
+
+variable "url_domain_name" {
+  type        = string
+  description = "Name of the domain provided by the user. This var will only be used if url_domain_addr is not empty"
+  default     = ""
+}
+
+variable "support_email" {
+  type        = string
+  description = "Email for users to contact with questions about their consent"
+  default = "<email>"
```

Comment:
NIT: same spacing here, probably in the other variable files too, assuming that you did not type out the the variable files.